app/process/watcher/evm/watcher.go

```diff
@@ -467,16 +502,11 @@ func (ew *Watcher) handleLockLog(eventLog *router.RouterLock, q qi.Queue) {
 		recipientAccount = common.BytesToAddress(eventLog.Receiver).String()
 	}
 
-	wrappedAsset := ew.mappings.NativeToWrapped(token, sourceChainId, targetChainId)
+	wrappedAsset := ew.assetsService.NativeToWrapped(token, sourceChainId, targetChainId)
 	if wrappedAsset == "" {
 		ew.logger.Errorf("[%s] - Failed to retrieve native asset of [%s].", eventLog.Raw.TxHash, eventLog.Token)
 		return
 	}
-	nativeAsset := ew.mappings.FungibleNativeAsset(sourceChainId, token)
-	if eventLog.Amount.Cmp(nativeAsset.MinAmount) < 0 {
-		ew.logger.Errorf("[%s] - Transfer Amount [%s] less than Minimum Amount [%s].", eventLog.Raw.TxHash, eventLog.Amount, nativeAsset.MinAmount)
-		return
-	}
```

Comment:
The check has to be done here otherwise using properAmount later might have a reduced value compared to the native token.